style.css

```diff
@@ -472,3 +478,116 @@ hr {
   width: 93%;
   position: absolute;
 }
+
+.section-form {
+  background: #6070ff;
+  border-top-left-radius: 60px;
+  margin-top: 30px;
+  background-image:
+    url(images/Rectangle-4.png),
+    url(images/Ellipse-3.png),
+    url(images/Ellipse-12.png);
+  background-position-x: 101%, 101%, 41%;
+  background-position-y: 5.3%, 9%, 9%;
+  background-repeat: no-repeat;
+}
+
+.form-div {
+  text-align: center;
+  margin-top: 80px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  padding-top: 90px;
+  color: #fff;
+}
+
+.form-div h3 {
+  font-weight: 700;
+  font-size: 50px;
+  line-height: 52px;
+}
+
+.form-div p {
+  display: flex;
+  align-items: center;
+  margin: 10px 20px;
+  font-weight: 200px;
+  font-size: 20px;
+  line-height: 20px;
+  text-align: center;
+  width: 70%;
+  height: 80%;
+  margin-left: 16%;
+}
+
+form {
+  outline: none;
+  width: 90%;
+  height: 90%;
+  margin-left: 20px;
+  margin-top: 10px;
+  background-image:
+    url(images/Rectangle-7.png),
+    url(images/Ellipse-5.png);
+  background-position-x: 88%, 81.9%;
+  background-position-y: 95%, 92%;
+  background-repeat: no-repeat;
+}
+
+form input {
+  margin: 10px;
+  padding: 10px;
+  outline: none;
+  width: 70%;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+}
+
+form textarea {
+  margin: 10px;
+  padding: 10px;
+  outline: none;
+  width: 70%;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+}
+
+.form-button {
+  text-align: center;
+  border-radius: 8px;
+  width: 30%;
+  height: 30%;
+  margin-top: 10px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-size: 16px;
+  font-weight: 150px;
+  background-color: #fff;
+  border: 1px solid #2230d2;
+  color: #6070ff;
+  margin-bottom: 15px;
+  letter-spacing: 0.03em;
+  cursor: pointer;
+}
+
+.form-button:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
+
+.form-button:active {
+  background-color: #2230d2;
+  color: #fff;
+}
+
+.form-button:disabled {
+  background-color: #ebebff;
+  color: #5e6c84;
+  border: 1px solid #5e6c84;
+}
+
+form img {
+  margin-bottom: 10px;
+  height: 3px;
+  border-radius: 8px;
+}
```

Comment:
- [ ] Good job with the layout of your project :+1: :100:, but I think it would be better if you stick to the Figma template as much as possible, this includes the font, colors, images, and spacing (margin/padding). 
  - The elements should be aligned using `Flexbox`
  - I think that the title of your form should be a little bit smaller :+1:
  - The `textarea` should be bigger :+1:  
  - The submit button should be aligned to the left :+1: 

**Your design**

![Screenshot from 2023-01-25 04-18-59](https://user-images.githubusercontent.com/105117832/214526040-048c2195-6b37-486e-ba8e-684cd522cf7d.png)

**Figma**

![Screenshot from 2023-01-25 04-19-41](https://user-images.githubusercontent.com/105117832/214526099-aaa4f3c4-2249-4748-8760-6851a518cd2c.png)